internal/configs/version1/nginx-plus.ingress.tmpl

```diff
@@ -260,9 +260,9 @@ server {
 		proxy_set_header Upgrade $http_upgrade;
 		proxy_set_header Connection $connection_upgrade;
 		{{- else}}
-		{{- if $.Keepalive}}proxy_set_header Connection "";{{end}}
+		{{- if $.Keepalive}}
```

Comment:
why does this change?

Reply:
There was formatting issues such as weird spacing and blank lines, can revert if needed